### ISSUE TITLE
UX - Standardize Picker Component Pattern

### ISSUE THREAD

**Issue by kellylein** [department-of-veterans-affairs/va-mobile-app#5470]
## Description 
As a Veteran using the mobile app, I want to have the experience of filtering things in the app be consistent, so that I understand how to filter and sort everywhere in the app.

As part of this story we want to:   
- look holistically at our Picker pattern to determine how we want them to work throughout the app.

## Acceptance Criteria 
-  [ ] Determine the pattern we want to use everywhere in the app for filter/sort of a list.
-  [ ] work with Product to create a FE implementation ticket/Epic to make the updates in the app

**Accessibility Requirements**
- [ ] AC 1
- [ ] AC 2
- [ ] AC 3

## Notes & Open Questions

- Dependencies/Roadblocks:
    - Any internal/external dependencies?
    - Test accounts needed?
- Does this require QA?
    
- Dev Notes: 

## Ticket Checklist

- [ ] Acceptance criteria defined
- [ ] Labels added (front-end, back-end, feature)
- [ ] Linked to an Epic

**Comment by timwright12** [comment]
@jessicawoodin @kellylein Design system repo?

**Comment by jessicawoodin** [comment]
@timwright12 Yes, I think we can transfer this to the design system repo. I can add this to my list when we get to patterns.
